modules/tinymce/src/core/main/ts/keyboard/DeleteBackspaceKeys.ts

```diff
@@ -43,9 +44,7 @@ const executeKeydownOverride = (editor: Editor, caret: Cell<Text>, evt: Keyboard
     { keyCode: VK.DELETE, action: MatchKeys.action(BlockBoundaryDelete.backspaceDelete, editor, true) },
     { keyCode: VK.BACKSPACE, action: MatchKeys.action(InlineFormatDelete.backspaceDelete, editor, false) },
     { keyCode: VK.DELETE, action: MatchKeys.action(InlineFormatDelete.backspaceDelete, editor, true) }
-  ], evt).each((_) => {
-    evt.preventDefault();
-  });
+  ], evt).each(mockInputEvents(editor, { inputType: 'deleteContentBackward' }, evt));
```

Comment:
Yeah `beforeinput` is definitely trickier since it involves a fair bit of refactoring to make it happen before the content changes and to make it preventable. Looking at the Zendesk ticket it does look like `input` was the primary issue so if needed we can most likely look at splitting this into 2 tasks: one for `beforeinput` and one for `input`.

Reply:
I'll drop the `beforeinput` from this one then, and just make an issue for it to be implemented as well at some point in the future.